src/vec3.rs

```diff
@@ -224,6 +227,7 @@ macro_rules! impl_vec3_float_methods {
         /// Given a unit-length vector return two other vectors that together form an orthonormal basis.
         /// That is, all three vectors are orthogonal to each other and are normalized.
         #[inline]
+        #[cfg(not(target_arch="spirv"))]
```

Comment:
How come these ones need to be disabled? Missing copysign?

Reply:
Yep, copysign isn't available in `num-traits` at the moment. I'm pushing for it to get merged but until then I needed to disable these functions.